Foundation/CGFloat.swift

```diff
@@ -615,174 +619,178 @@ public func %=(lhs: inout CGFloat, rhs: CGFloat) {
 // tgmath
 //===----------------------------------------------------------------------===//
 
+#if os(Android)
+import Glibc
+#endif
+
 @_transparent
 public func acos(_ x: CGFloat) -> CGFloat {
-    return CGFloat(acos(x.native))
+    return CGFloat(acos(x.double))
```

Comment:
What is going on here? Doesn't this change the result of `acos(_: CGFloat)` for all 32-bit platforms?